### PR TITLE
Reserve extension 286 for wl_acquire_display

### PR DESCRIPTION
**Pull request by ddevault**
See #1001

### CODE REVIEW

File: xml/vk.xml [hunk 1]
```diff
@@ -11222,6 +11222,12 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="0"                                             name="VK_EXT_EXTENSION_285_SPEC_VERSION"/>
                 <enum value="&quot;VK_EXT_extension_285&quot;"              name="VK_EXT_extension_285"/>
             </require>
+        </extension>
+		<extension name="VK_EXT_extension_286" number="286" type="instance" author="MESA" contact="Drew DeVault <sir@cmpwn.com>" supported="disabled">
```
Comment (jekstrand) [inline xml/vk.xml]: I think your text editor had a minor issue here and the indentation got messed up.  We use spaces, not tabs.

**Comment by ddevault** [comment]
Whoops, good catch.

File: xml/vk.xml (hunk 1 above)
Comment (jekstrand) [reply xml/vk.xml]: Looks good to me now.

File: xml/vk.xml [hunk 2]
```diff
@@ -11223,5 +11223,11 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_EXT_extension_285&quot;"              name="VK_EXT_extension_285"/>
             </require>
         </extension>
+        <extension name="VK_EXT_extension_286" number="286" type="instance" author="MESA" contact="Drew DeVault <sir@cmpwn.com>" supported="disabled">
```
Comment (krOoze) [inline xml/vk.xml]: Shouldn't it have `MESA` prefix then? `MESA` seem defined in XML as tag.

Comment (ddevault) [reply xml/vk.xml]: I think EXT is appropriate and MESA is not. I wrote that before I understood the meaning of the field and never went back to change it. Just pushed a fixed version now.

Comment (jekstrand) [reply xml/vk.xml]: It also really doesn't matter when just reserving an extension number.  We can do "the right thing" when the real version of the extension rolls out.

Comment (krOoze) [reply xml/vk.xml]: Well yea, though it might be annoying to refactor the whole extension spec with proper tag.

**Comment by ddevault** [comment]
Thanks!

**Comment by oddhack** [comment]
BTW, the angle brackets around the email address are invalid XML syntax - I didn't notice this until I had cherry-picked the commit into our internal gitlab server where we run CI tests. I will be pushing a spec update shortly which just removes the brackets. Also, we prefer to use github IDs here instead of email addresses.

**Comment by ddevault** [comment]
Thanks for catching that! I would prefer to use my email than my GitHub username, though.

**Pull request merged**